### PR TITLE
update oidc provider config attribute descriptions

### LINKED ISSUE #25193: Two inaccurate descriptions and one formatting problem in openidConnectProvider
**Describe the bug**  
On https://openliberty.io/docs/latest/reference/config/openidConnectProvider.html

The type for `jwkRotationTime` says `A period of time with minute precision`.  However, the description says that you can set the time in minutes or hours.  Neither says what will happen if you give no units.  I suppose minutes?

I think the same can be said for `idTokenLifetime`, which says:

> `Time that ID token is valid (seconds). Specify a positive integer followed by a unit of time, which can be hours (h), minutes (m), or seconds (s). For example, specify 30 seconds as 30s. You can include multiple values in a single entry. For example, 1m30s is equivalent to 90 seconds.`

If what really happens is: "_if you enter just 30, it is 30 seconds, but you can also use 30s for 30 seconds and 30m for 30 minutes_", etc, should the description be:

> `Time that ID token is valid (seconds). You can also specify a positive integer followed by a unit of time, which can be hours (h), minutes (m), or seconds (s). For example, specify 30 seconds as 30s. You can include multiple values in a single entry. For example, 1m30s is equivalent to 90 seconds.`

If that is what is the accurate description for `idTokenLifetime`, then is this the accurate description for `jwkRotationTime`?

> `Amount of time after which a new JWK will be generated (minutes). You can also pecify a positive integer followed by a unit of time, which can be hours (h) or minutes (m). For example, specify 30 minutes as 30m. You can include multiple values in a single entry. For example, 1h30m is equivalent to 90 minutes.`

Please add the minimum value if there is one.  For both entries.

While you're in there, notice this is in the description for `jwkEnabled`:

> `https://&lt;host_name&gt;:&lt;port_number&gt;/oidc/endpoint/&lt;provider_id&gt;/jwk`

### PR DESCRIPTION
**Pull request by arunavemulapalli**
resolves #25193

### CODE REVIEW

**Comment by arunavemulapalli** [comment]
#build

File: dev/com.ibm.ws.security.openidconnect.server/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -174,13 +174,13 @@ thirdPartyIDTokenClaims=Extra claims from a third-party ID token
...
+jwkEnabled.desc=When JWK is enabled, the OpenID Connect provider dynamically generates key pairs for signing JWT tokens that it creates. To validate the signature, the client can retrieve the key from the JWK endpoint, which has the format https://\<host_name\>:\<port_number\>/oidc/endpoint/\<provider_id\>/jwk. When this attribute is false, the provider will use the key that is specified by the keyAliasName attribute to sign the JWT. The provider will still make that key available as a JWK from the JWK endpoint if this attribute is false.
 
 idTokenCacheEnabled=Cache ID tokens
 idTokenCacheEnabled.desc=Enable caching to save ID tokens in the database and in-memory cache.
 
 jwkRotationTime=JWK rotation time
-jwkRotationTime.desc=Amount of time after which a new JWK will be generated.
+jwkRotationTime.desc=Amount of time after which a new JWK will be generated (minutes). You can also specify a positive integer followed by a unit of time, which can be hours (h) or minutes (m). For example, specify 30 minutes as 30 or 30m. You can include multiple values in a single entry. For example, 1h30m is equivalent to 90 minutes.
```
Comment (ramkumar-k-9286) [inline dev/com.ibm.ws.security.openidconnect.server/resources/OSGI-INF/l10n/metatype.properties]:
```suggestion
jwkRotationTime.desc=Amount of time after which a new JWK is generated (minutes). You can also specify a positive integer followed by a unit of time, which can be hours (h) or minutes (m). For example, specify 30 minutes as 30 or 30m. You can include multiple values in a single entry. For example, 1h30m is equivalent to 90 minutes.
```

**Review by ayoho — DISMISSED** [review]

**Comment by arunavemulapalli** [comment]
#libby

**Comment by arunavemulapalli** [comment]
#build

**Comment by posmith** [comment]
L2 message review not required

**Review by ayoho — APPROVED** [review]

**Pull request merged**